README.md

```diff
@@ -66,17 +66,17 @@ And then you can import the CSS file found in `dist/sbuttons.css` or the LESS fi
 
 ### Modifying Button Colors
 
-To modify button colors, import `src/sbuttons.less` in your Less file, then make changes to the variables after the import.
+To modify button colors, import `src/_variables.less` in your Less file, then make changes to the variables after the import.
```

Comment:
Don't make changes here

Reply:
should i revert back those changes?

Reply:
yes

Reply:
but this was mentioned in issue that " You should also change the link in the README for sbuttons.less under Modifying Button Colors to the new variables.less file and change the text as well to "_variables.less" "

Reply:
oh got it

Reply:
I did the changes as you requested.